rotkehlchen/chain/ethereum/modules/makerdao/cache.py

```diff
@@ -246,15 +246,15 @@ def update_ilk_registry(
         # TODO: We should introduce a key-value cache with key being primary key too
         questionmarks = ','.join('?' * len(write_tuples))
         write_cursor.execute(
-            f'DELETE from general_cache WHERE key IN ({questionmarks})',
+            f'DELETE from unique_cache WHERE key IN ({questionmarks})',
```

Comment:
As you can see from the comments and the code here this is not needed now that you are using the correct cache. These 3 sql statements should be an `INSERT OR REPLACE`

Reply:
oww okay.